packages/web/components/Landing/sections/Build.tsx

```diff
@@ -0,0 +1,50 @@
+import { Box, Container, Text } from '@metafam/ds';
+import BackgroundImage from 'assets/landing/build-background.png';
+
+export const Build: React.FC = () => (
+  <Box
+    width="100%"
+    minHeight="100%"
+    maxHeight="100%"
+    backgroundImage={`url(${BackgroundImage})`}
+    bgPosition="center"
+    bgSize="cover"
+  >
+    <Container
+      width="100%"
+      height="100vh"
+      display="flex"
+      flexDirection="column"
+      justifyContent="center"
+      maxWidth="100%"
+      pl={{ base: '0', md: '137px' }}
```

Comment:
Hey there's still places where px is used, which can be replaced with rem equivalents, like this one.